packages/rke2-kube-proxy-1.21/charts/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: rke2-kube-proxy
 description: Install Kube Proxy.
-version: v1.21.12-rke2r2-build20220428
+version: v1.21.12-rke2r3-build20220505
 appVersion: v1.21.12-rke2r2
```

Comment:
Update this line as well.

Reply:
Are your refering to changing line 5 to be:
`appVersion: v1.21.12-rke2r3`